src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -713,6 +713,17 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
     }
 
     /**
+     * Create and return and un-saved model instance.
+     *
+     * @param  array  $attributes
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function make(array $attributes = [])
```

Comment:
Unless I'm mistaken, `__callStatic` only works for inaccessible methods, i.e. `private`/`protected` and non-existent methods so to leverage that, you'd likely want to make the method `protected` if going down that path. 

Otherwise, yes, as @JosephSilber suggested, it ought to be static. I'm not even certain how it's working as is 🔮